llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -27,6 +27,64 @@
 
 using namespace llvm;
 
+namespace {
+
+class CFISaveRegisterEmitter {
+  MachineFunction &m_MF;
```

Comment:
LLVM doesn't use `m_` and we don't generally have underscores in variable names.